realm/realm-library/src/androidTest/java/io/realm/MutableRealmIntegerTests.java

```diff
@@ -346,8 +355,93 @@ public void copyFromIsNotLive() {
+        realm.beginTransaction();
+        obj = realm.createObjectFromJson(MutableRealmIntegerTypes.class, json);
+        realm.commitTransaction();
+        assertTrue(obj.columnNullableMutableRealmInteger.isNull());
+
+        json = new JSONObject();
+        json.put("columnNonNullableMutableRealmInteger", JSONObject.NULL);
+        realm.beginTransaction();
+        try {
+            obj = realm.createObjectFromJson(MutableRealmIntegerTypes.class, json);
+            fail("Attempt to set @Required Mutable Realm Integer null, from JSON, should fail");
+        } catch (IllegalArgumentException ignore) {
+            checkException(ignore, "is not nullable");
+        }
+        realm.commitTransaction();
+    }
+
+    @Test
+    public void testStream() throws IOException {
+        Context context = InstrumentationRegistry.getTargetContext();
+
+        InputStream in = TestHelper.loadJsonFromAssets(context, "empty.json");
+        realm.beginTransaction();
+        MutableRealmIntegerTypes obj = realm.createObjectFromJson(MutableRealmIntegerTypes.class, in);
+        realm.commitTransaction();
+        assertTrue(obj.columnNullableMutableRealmInteger.isNull());
+
+        in = TestHelper.loadJsonFromAssets(context, "mutablerealminteger-long.json");
+        realm.beginTransaction();
+        obj = realm.createObjectFromJson(MutableRealmIntegerTypes.class, in);
+        realm.commitTransaction();
+        assertEquals(Long.valueOf(8589934592L), obj.columnNullableMutableRealmInteger.get());
+
+        in = TestHelper.loadJsonFromAssets(context, "mutablerealminteger-int.json");
+        realm.beginTransaction();
+        obj = realm.createObjectFromJson(MutableRealmIntegerTypes.class, in);
+        realm.commitTransaction();
+        assertEquals(Long.valueOf(22), obj.columnNullableMutableRealmInteger.get());
+
+        in = TestHelper.loadJsonFromAssets(context, "mutablerealminteger-null.json");
+        realm.beginTransaction();
+        obj = realm.createObjectFromJson(MutableRealmIntegerTypes.class, in);
+        realm.commitTransaction();
+        assertTrue(obj.columnNullableMutableRealmInteger.isNull());
+
+        in = TestHelper.loadJsonFromAssets(context, "mutablerealminteger-required-null.json");
+        realm.beginTransaction();
+        try {
+            obj = realm.createObjectFromJson(MutableRealmIntegerTypes.class, in);
+            fail("Attempt to set @Required Mutable Realm Integer null, from JSON, should fail");
+        } catch (IllegalArgumentException ignore) {
+            checkException(ignore, "is not nullable");
+        }
+        realm.commitTransaction();
+    }
+
+    private void checkTransactionException(Exception e) {
+        checkException(e, "only be done from inside a transaction");
+    }
+
+    private void checkException(Exception e, String expected) {
+        System.out.println("Expecting: " + expected + ", got: " + e);
+        assertTrue(e.getMessage().contains(expected));
```

Comment:
We don't need to use `System.out.println()` here.
These two lines should be `assertTrue("Expecting: " + expected + ", got: " + e, e.getMessage().contains(expected));`.